### ISSUE TITLE
Can't specify to run Up with a specific VagrantFile

### ISSUE THREAD

**Issue by SavaNDragos** [agc93/Cake.Vagrant#2]
In some cases it would be nice to be able to just call Vagrant.Up(VagrantFileLocation).
It could posible support this but i didn't manage to see how.

**Comment by agc93** [comment]
Hmmm, that's a good point. Do you know if the CLI supports this natively? I can't find it in the [online docs](https://www.vagrantup.com/docs/cli/up.html) but I might be missing something.

Another alternative is the [Cake.DoInDirectory](https://github.com/pitermarx/Cake.DoInDirectory) addin. 

I will investigate if supporting this "natively" might be easy enough.

**Comment by SavaNDragos** [comment]
The CLI doesn't support that, so you need to have WorkingDirectory set to Vagrant File Directory. 
You could do it by adding one more parameter to the Up command : 
public void Up(string name, string vagrantFileDirectory, Action<VagrantUpSettings> configure) 
or by adding it as a parameter in the VagrantUpSettings and setting after that Settings.WorkingDirectory = vagrantFileDirectory;
 Run(Settings, args);

**Comment by agc93** [comment]
I've just pushed a new experimental version 28 to [NuGet](https://www.nuget.org/packages/Cake.Vagrant/0.3.0-unstable0028). You can try it out using the prerelease flag:

```
#addin nuget:?package=Cake.Vagrant&prerelease
```

Basically, I've shamelessly stolen @philo's `FromPath()` implementation from `cake-npm` and added it to the runner. Usage looks like:

```
Vagrant.FromPath("./path/to/directory").Up();
```

Would that meet your needs?

**Comment by SavaNDragos** [comment]
It is perfect.

**Comment by agc93** [comment]
Fixed in [0.3.0 release](https://github.com/agc93/Cake.Vagrant/commit/4acbec1a6f043c400b94f8b2bc5bcb48797f1b15).

Now available on [the NuGet Gallery](https://www.nuget.org/packages/Cake.Vagrant/0.3.0)
